scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -756,7 +756,7 @@ private[scalatest] class DashboardReporter(directory: String,
   // so nested events within the TestRecord have been removed.)
   //
   class TestRecord(startEvent: TestStarting) {
-    var endEvent: Event = null
+    var endEvent: Event | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.